descriptors/ModuleDescriptor-template.json

```diff
@@ -717,19 +717,22 @@
       "description": "Delete an existing Invoice"
     },
     {
-      "permissionName": "invoice.item.approve",
+      "permissionName": "invoice.item.approve.execute",
       "displayName": "Invoice - approve an existing Invoice",
-      "description": "Approve an existing Invoice"
+      "description": "Approve an existing Invoice",
+      "replaces": "invoice.item.approve"
     },
     {
-      "permissionName": "invoice.item.pay",
+      "permissionName": "invoice.item.pay.execute",
       "displayName": "Invoice - pay an existing Invoice",
-      "description": "Pay an existing Invoice"
+      "description": "Pay an existing Invoice",
+      "replaces": "invoice.item.pay"
     },
     {
-      "permissionName": "invoice.item.cancel",
+      "permissionName": "invoice.item.cancel.execute",
       "displayName": "Invoice - cancel an existing Invoice",
-      "description": "Cancel an existing Invoice"
+      "description": "Cancel an existing Invoice",
+      "replaces": ["invoice.item.cancel"]
```

Comment:
Is there some difference between getting in array brackets or not, for example for "invoice.item.pay" we don't have them

Reply:
good catch, it should be always array

Reply:
commited changes